src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ProfilesController.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/**
+ * 2007-2018 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Controller\Admin\Configure\AdvancedParameters;
+
+use PrestaShop\PrestaShop\Core\Search\Filters\ProfilesFilters;
+use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
+use PrestaShopBundle\Security\Annotation\AdminSecurity;
+use PrestaShopBundle\Security\Annotation\DemoRestricted;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Class ProfilesController is responsible for displaying the
+ * "Configure > Advanced parameters > Team > Profiles" page.
+ */
+class ProfilesController extends FrameworkBundleAdminController
+{
+    /**
+     * Show profiles listing page.
+     *
+     * @AdminSecurity("is_granted('read', request.get('_legacy_controller'))")
+     *
+     * @Template("@PrestaShop/Admin/Configure/AdvancedParameters/Profiles/profiles.html.twig")
```

Comment:
Check naming convention :+1: 

```suggestion
     * @Template("@PrestaShop/Admin/Configure/AdvancedParameters/Profiles/index.html.twig")
```